pom.xml

```diff
@@ -214,39 +214,13 @@
                     </executions>
                 </plugin>
                 <plugin>
-                    <artifactId>maven-dependency-plugin</artifactId>
-                    <version>2.10</version>
-                    <executions>
-                        <execution>
-                            <id>unpack-license</id>
-                            <phase>generate-sources</phase>
-                            <goals>
-                                <goal>unpack</goal>
-                            </goals>
-                            <configuration>
-                                <artifactItems>
-                                    <artifactItem>
-                                        <!-- includes common copyright headers for sources -->
-                                        <groupId>org.kaazing</groupId>
-                                        <artifactId>community.license</artifactId>
-                                        <type>jar</type>
-                                    </artifactItem>
-                                </artifactItems>
-                                <overWriteReleases>true</overWriteReleases>
-                                <overWriteSnapshots>true</overWriteSnapshots>
-                                <failOnMissingClassifierArtifact>true</failOnMissingClassifierArtifact>
-                            </configuration>
-                        </execution>
-                    </executions>
-                </plugin>
-                <plugin>
                     <!-- To update the license, do: -->
-                    <!--     mvn com.mycila:license-maven-plugin:RELEASE:format -DskipITs -DskipTests -->
+                    <!--     mvn license:format -->
```

Comment:
Ah, makes sense.

Btw, the `RELEASE` part is incorrect because it overrides the version defined by the POM.

It should be `mvn com.mycila:license-maven-plugin:format` instead.